### ISSUE TITLE
Add space model tests

### ISSUE THREAD

**Issue by hernanmd** [cormas/cormas#455]
This class (CMSpaceModel) is core to CORMAS and largely untested.

**Comment by SergeStinckwich** [comment]
Agree with you. Maybe we can try to organize a pair programming session to understand how it works and to add more tests?

**Comment by tomooda** [comment]
I once tried to make test cases for the space model but I couldn't because it was too complex to understand how each method should work at all. ;-(
I think pair programming session is a good idea. Maybe at the next meeting?

**Comment by hernanmd** [comment]
Yes please, there are some methods which are costly to understand and use!

**Comment by hernanmd** [comment]
I put this on-hold until #465 is somewhat completed.

**Comment by hernanmd** [comment]
Was this really implemented or is it closed due to lack of activity?
